database/couchdb.js

```diff
@@ -10,7 +10,7 @@ const seed = async (database) => {
   const photos = await listPhotos();
   let docs = [];
 
-  for (let run = 0; run < 1000; run++) {
+  for (let run = 0; run < 10000; run++) {
```

Comment:
Doesn't this need to be 10M? Or were you just making sure it ran?

Reply:
there are a 1000 different photos in s3 bucket.. this script amounts to 10M records in DB.. 1000 for this outer loop only amounted in 1M